src/device/usbd_control.c

```diff
@@ -81,7 +82,7 @@ bool tud_control_status(uint8_t rhport, tusb_control_request_t const * request)
   return _status_stage_xact(rhport, request);
 }
 
-// Transfer an transaction in Data Stage
+// Queue an transaction in Data Stage
```

Comment:
s/an/a/

Reply:
damn, why English uses different words for the same thing !!! :sweat: